dvc/project.py

```diff
@@ -143,6 +143,14 @@ def _check_output_duplication(self, outs):
                         stages = [o.stage.relpath, out.stage.relpath]
                         raise OutputDuplicationError(o.path, stages)
 
+    def _check_outs_duplication(self, outs):
```

Comment:
There's already a `_check_output_duplication` function, that checks if the file already exists on the Stage.
You can find the related test right here: https://github.com/mroutis/dvc/blob/6d848cfe64b43ee506624c709513045c7cb4abcc/tests/test_run.py#L39-L45

The test previously ran the `dvc` command with `out` specified as an output, and stored the resulting Dvcfile on a file named `out.dvc`. Then, the command is called again, but with different Dvcfile (`duplicateout.dvc`) this time.

It would be confusing to have a similar name for a function that does something different: check if the output was specified more than once as an argument in the CLI, however, something more useful would be checking for duplicated arguments in general, not just for duplicated output arguments. I'll change the issue to reflect this change